concourse/pipelines/build-images.yml

```diff
@@ -0,0 +1,89 @@
+---
+definitions:
+
+resources:
+  - icon: github
+    name: smokey-commit
+    type: git
+    source:
+      uri: git@github.com:alphagov/smokey.git
+      branch: main
+      private_key: |
+        ((govukci_private_key))
+
+  - icon: github
+    name: govuk-infrastructure
+    type: git
+    source:
+      uri: git@github.com:alphagov/govuk-infrastructure.git
+      branch: main
+      private_key: |
+        ((govukci_private_key))
+
+  - name: smokey-image
+    type: registry-image
+    icon: docker
+    source:
+      repository: govuk/smokey
+      tag: latest
+      username: ((docker_hub_username))
+      password: ((docker_hub_authtoken))
+
+  - name: smokey-version
+    type: semver
+    source:
+      driver: s3
+      access_key_id: ((readonly_access_key_id))
+      secret_access_key: ((readonly_secret_access_key))
+      session_token: ((readonly_session_token))
+      bucket: ((readonly_private_bucket_name))
+      key: smokey-version
+      region_name: eu-west-2
+      initial_version: '1.0.0'
+
+jobs:
+  - name: update-pipeline
+    plan:
+    - get: govuk-infrastructure
+      trigger: true
+    - file: govuk-infrastructure/concourse/pipelines/build-images.yml
+      set_pipeline: build-images
+
+  - name: smokey
+    plan:
+    - in_parallel:
+      - get: smokey-commit
+        trigger: true
+      - get: smokey-version
+        params:
+          bump: minor
+    - task: build-image
+      privileged: true
```

Comment:
Do we need `privileged: true` here?

Reply:
Yeah, though perhaps not forever: https://github.com/vito/oci-build-task#oci-build-task

> A stretch goal of this is to support running without privileged: true, though it currently still requires it.

Reply:
"A stretch goal of this is to support running without privileged: true, though it currently still requires it." so... yes:

https://github.com/vito/oci-build-task

That's no problem then :)